### DISCUSSION TITLE
"Continue series" to "on deck"

### DISCUSSION THREAD

**Discussion by ajurna** [advplyr/audiobookshelf#3627] [Ideas]
So i love the continue series functionality and would love to add to it. 
I would like to be able to add audiobooks i want to listen to next to it. Also to remove ones i don't want to finish. 
I'll often get a new book and then forget about it. Going through my library can be a bit of a chore as it's quite large.

**Comment by nichwall** [comment]
Customizing the home screen shelves is planned. https://github.com/advplyr/audiobookshelf/issues/1273
